example/deployment/local-k8s/README.md

```diff
@@ -0,0 +1,72 @@
+# Kubernetes Example
+
+This example demonstrates different users querying an avro file over a REST api running locally in kubernetes containers.
+
+The example runs several different queries by the different users, with different purposes. When you run the example you will see the data has been redacted in line with the rules.  
+For an overview of the example see [here](../../README.md)
+
+### Prerequisites for running in kubernetes 
+As well as docker, this example also requires Kubernetes
+
+To run the example locally in docker containers (under kubernetes) follow these steps (from the root of the project):
+
+1. Compile the code:
+    ```bash
+    mvn clean install -P example
+    ```
+
+2. Start the REST services:
+    If you have made changes to the code since you last built the docker containers then you will need to run:
+    
+    *NOTE* This will clean all stopped docker services, requiring them to be rebuilt when you next want to start them up. 
+    ```bash
+     ./example/deployment/local-docker/bash-scripts/dockerCleanSystem.sh
+    ```
+
+    Then you can start up the docker containers to create the docker images:
+    ```bash
+     ./example/deployment/local-docker/bash-scripts/dockerComposeCreateOnly.sh
+    ```
+    
+    Then you can start the kubernetes cluster:
+    ```bash
+    ./example/deployment/local-k8s/bash-scripts/buildServices.sh
+    ```
+
+    You can check the pods are available:    
+    ```bash
+    kubectl get pods
+    ```
+
+    After a while you should see the liveness and readiness probes indicating all is well - see the example below:
+    ```bash    
+    Palisade/example/deployment  gh-335-local-k8s ✗                                                                 46m ◒
+    ▶ kubectl get pods
+    NAME                                READY     STATUS      RESTARTS   AGE
+    config-service-b8fb57cbc-z6pgg      1/1       Running     0          8m
+    configure-services-z8p7w            0/1       Completed   0          8m
```

Comment:
This will need updating to reflect the configure-example job having completed as well